### PR TITLE
Allow the map to have a projection other than 3857

### LINKED ISSUE #172: Allow map to be in projections other than 3857
Allow the map to be set to projections other than 3857. While 3857 is a good default, there are many instances where a custom projection would be better suited to the data layers being presented. For us that would be EPSG:27700 (British National Grid), but for others that might be 4326, or a country specific coordinate system.

This should be settable on a version by version basis, with a default of 3857.

### PR DESCRIPTION
**Pull request by RobQuincey-DC**
This PR adds the ability for maps to be in projections other than EPSG:3857 (Spherical Mercator).

To facilitate this, a new 'Projections' table has been added to the database, and **all versions must now have a projection defined**. The Projections table (and linked management UI) allows you to add supported projections using WKT or Proj4 format. Built in projections (3857, 4326,  4269, 900913) do not need the WKT/Proj4 string added.

Layers will attempt to be rendered in the projection of the map, if there are layers which can not be rendered in the projection of the map, then they will need their projection explicitly defined. This is no different to how it works at the moment, except that if you want a layer available in two different maps in different projections, and ONE of the map projections is not supported by the layer, you will need to create two layers or define a compatible projection (such as EPSG:3857). This is unlikely to crop up often unless you have particularly obscure projections in use.

Versions can have multiple projections applied, but only one will be used for actual map rendering (by applying the 'IsDefaultMapProjection' boolean or appropriate checkbox in management UI), the rest will be used for rendering coordinates on screen (with the default being set using the 'IsDefaultViewProjection' boolean or appropriate checkbox in management UI).

Additional tweaks:
- Requests to the SignalR broadcasthub endpoint have been filtered out of telemetry
- UI and URLs of 'System' section of management UI have been updated for consistency

Closes #172

### CODE REVIEW

File: GIFrameworkMaps.Web/Scripts/Map.ts [hunk 1]
```diff
@@ -1123,12 +1171,14 @@ export class GIFWMap {
           if (source instanceof TileWMS || source instanceof ImageWMS) {
             const view = this.olMap.getView();
             const viewport = this.olMap.getViewport();
+            //version is forced to 1.1.0 to get around lat/lon flipping issues
```
Comment (RobQuincey-DC) [inline GIFrameworkMaps.Web/Scripts/Map.ts:1174]: See if we can fix this properly

File: GIFrameworkMaps.Web/Scripts/Map.ts [hunk 2]
```diff
@@ -228,25 +245,25 @@ export class GIFWMap {
...
     let startMinZoom: number = 0;
     const startBasemap = this.config.basemaps.find((b) => b.isDefault);
     if (startBasemap !== null) {
-      startExtent = [
-        startBasemap.bound.bottomLeftX,
-        startBasemap.bound.bottomLeftY,
-        startBasemap.bound.topRightX,
-        startBasemap.bound.topRightY,
-      ];
+      //startExtent = [
```
Comment (RobQuincey-DC) [inline GIFrameworkMaps.Web/Scripts/Map.ts:253]: Remove comments!

File: GIFrameworkMaps.Web/Views/ManagementProjection/Delete.cshtml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿@model GIFrameworkMaps.Data.Models.Projection
+@{
+    ViewData["Title"] = $"Delete projection '{Model.Name}'";
+}
+
+<h1>@ViewData["Title"]</h1>
+<div class="row mt-2">
+    <h2>Are you sure you want to delete this projectionn?</h2>
```
Comment (RobQuincey-DC) [inline GIFrameworkMaps.Web/Views/ManagementProjection/Delete.cshtml:8]: Spelling

File: GIFrameworkMaps.Web/Views/ManagementSystem/Index.cshtml [hunk 4]
```diff
@@ -8,7 +8,7 @@
 <h1>@ViewData["Title"]</h1>
 <p class="lead">Check and update system settings and cache</p>
 
-<div class="row">
+@* <div class="row">
```
Comment (RobQuincey-DC) [inline GIFrameworkMaps.Web/Views/ManagementSystem/Index.cshtml:11]: Remove commented code

File: GIFrameworkMaps.Web/Scripts/Map.ts (hunk 1 above)
Comment (RobQuincey-DC) [reply GIFrameworkMaps.Web/Scripts/Map.ts:1168]: Very minor issue, most servers handle 1.1.0 so will put on backlog

**Review by VulpesFerrilata — APPROVED** [review]
All tested and working 🌍

**Review by VulpesFerrilata — APPROVED** [review]

**Pull request merged**